katas/content/complex_arithmetic/complex_addition/solution.md

```diff
@@ -0,0 +1,14 @@
+**Inputs:**
```

Comment:
We don't need to repeat the statement in the solution, it shows up on the site above the code field, just the text after the **Solution:** header. (Jupyter notebook workbooks were separate files, so they repeated statement)

Reply:
Removed from all solutions.